napari/_qt/qt_main_window.py

```diff
@@ -132,6 +134,11 @@ def _add_file_menu(self):
         open_images.setStatusTip('Open image file(s)')
         open_images.triggered.connect(self.qt_viewer._open_images)
 
+        open_stack = QAction('Open image series as stack...', self._qt_window)
+        open_stack.setShortcut('Ctrl+Alt+O')
+        open_stack.setStatusTip('Open image files')
+        open_stack.triggered.connect(self.qt_viewer._open_images_as_stack)
```

Comment:
Here I think `open image file(s)` should maybe be renamed to `open image file(s) as layers...`, because to me, stack is the assumption...

Reply:
i need clarification, there are two items in the menu (ignoring the statusTip for a moment): 
1. `Open image(s)...`
2. `Open image series as stack...`

I think there should be one very simple `Open image(s)` option, which is the first one, which just takes a path, makes no assumptions about it, and adds it to the viewer.  It allows choosing multiple files, but no requirement/assumption about their shapes is made, they each make a new layer.  I don't love renaming that `open image file(s) as layer(s)` because that assumes some napari knowledge about what a layer _is_ ... there should just be an "open file" option and this is it.  The second option imposes requirements on the images (i.e. that they all be the same dimensions), and then builds a 3D stack from 2D images...

Are you saying that, to you, the assumption is that if you provide two file paths, they are usually meant to be stacked on top of each other?  And you want the second function to be the default one?

Reply:
> Are you saying that, to you, the assumption is that if you provide two file paths, they are usually meant to be stacked on top of each other? And you want the second function to be the default one?

Yes, that is what I was saying, *but*, you've convinced me otherwise. =) Well, I wasn't making judgment on the default, just saying that whichever default you picked, it should be documented in the menu name. Anyway, carry on... This problem will be much smaller when we add the "combine layers into stack" function. =)